gherkin/java/src/main/java/gherkin/GherkinLanguageConstants.java

```diff
@@ -1,10 +1,15 @@
 package gherkin;
 
+import java.util.regex.Pattern;
+
 public interface GherkinLanguageConstants {
     String TAG_PREFIX = "@";
     String COMMENT_PREFIX = "#";
     String TITLE_KEYWORD_SEPARATOR = ":";
+    Pattern TITLE_KEYWORD_SEPARATOR2 = Pattern.compile(":|：");
```

Comment:
This change must be ported to all other Gherkin implementations (Go, C, Ruby, JavaScript etc) before it can be merged.

Reply:
should we introduce a constants TITLE_KEYWORD_SEPARATOR_LENGTH? since pattern have no length.